### ISSUE TITLE
neotest adapter: the argument '--no-run' cannot be used multiple times

### ISSUE THREAD

**Issue by grihabor** [mrcjkb/rustaceanvim#357]
### Have you read the docs and searched existing issues?

- [X] I have read [the README's troubleshooting guide](https://github.com/mrcjkb/rustaceanvim/blob/master/README.md#troubleshooting).
- [X] I have searched [existing issues](https://github.com/mrcjkb/rustaceanvim/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
- [X] I have searched [existing discussions](https://github.com/mrcjkb/rustaceanvim/discussions).
- [ ] I have searched the existing issues of other plugins related to this issue (if applicable).
- [ ] I have searched the [existing rust-analyzer issues](https://github.com/rust-lang/rust-analyzer/issues?q=is%3Aissue+) (if applicable).
- [X] I can reproduce the issue with stable Neovim (not a distribution or nightly) and the [minimal config](https://github.com/mrcjkb/rustaceanvim/blob/master/troubleshooting/minimal.lua).

### Neovim version (nvim -v)

v0.9.4

### Operating system/version

Ubuntu 22.04

### Output of :checkhealth rustaceanvim

```console
==============================================================================
rustaceanvim: require("rustaceanvim.health").check()

Checking for Lua dependencies ~
- OK [mfussenegger/nvim-dap](https://github.com/mfussenegger/nvim-dap) installed.

Checking external dependencies ~
- OK rust-analyzer: found rust-analyzer 1.75.0 (82e1608 2023-12-21)
- OK Cargo: found cargo 1.75.0 (1d8b05cdd 2023-11-20)
- OK rustc: found rustc 1.75.0 (82e1608df 2023-12-21)
- OK debug adapter: found codelldb 

Checking config ~
- OK No errors found in config.

Checking for conflicting plugins ~
- OK No conflicting plugins detected.

Checking for tree-sitter parser ~
- OK tree-sitter parser for Rust detected.
```

### How to reproduce the issue

```console
Open a rust project, navigate to any test, set a breakpoint, then:

lua require("neotest").run.run({strategy = "dap"})
```
Test should stop on a breakpoint, then
```
lua require("neotest").run.stop()
```
Then run the test again:
```
lua require("neotest").run.run({strategy = "dap"})
```
```


### Expected behaviour

Test stops on the breakpoint and keeps waiting.

### Actual behaviour

Throws an error:
```bash 
Command: cargo test --no-run --no-run --no-run --no-run --no-run --no-run --no-run --package pyny_index_tests --test integration_test --message-format=json
stdout: error: the argument '--no-run' cannot be used multiple times
```

### The minimal config used to reproduce this issue.

```Lua
Lazy.nvim rustaceanvim config:

return {
    "mrcjkb/rustaceanvim",
    version = "^4",
    ft = { "rust" },
    config = function()
        vim.g.rustaceanvim = function()
            local extension_path = vim.env.HOME .. "/.nix-profile/share/vscode/extensions/vadimcn.vscode-lldb/"
            local codelldb_path = extension_path .. "adapter/codelldb"
            local liblldb_path = extension_path .. "lldb/lib/liblldb.so"

            local cfg = require("rustaceanvim.config")
            return {
                on_attach = require("config.lsp"),
                dap = {
                    adapter = cfg.get_codelldb_adapter(codelldb_path, liblldb_path),
                },
            }
        end
    end,
}
```

Install codelldb via (nix) home manager:
```
    pkgs.vscode-extensions.vadimcn.vscode-lldb
```
```

**Comment by mrcjkb** [comment]
Hey :wave: 

Thanks for the detailed report :pray: 
A fix is on the way.

**Comment by grihabor** [comment]
Thank you for a fast fix! :+1:

**Comment by grihabor** [comment]
@mrcjkb I've just checked the latest version, now I'm getting
```
rustaceanvim: ...re/nvim/lazy/rustaceanvim/lua/rustaceanvim/overrides.lua:69: attempt to call field 'list_contains' (a nil value)
```
I think it's related to the change in https://github.com/mrcjkb/rustaceanvim/pull/358

**Comment by mrcjkb** [comment]
> @mrcjkb I've just checked the latest version, now I'm getting
> 
> ```
> rustaceanvim: ...re/nvim/lazy/rustaceanvim/lua/rustaceanvim/overrides.lua:69: attempt to call field 'list_contains' (a nil value)
> ```
> 
> I think it's related to the change in #358

Oops, I was using a neovim nightly API call.
Fixed in version 4.22.4.
